### PR TITLE
Update the header to match the style of tjcog.org

### LINKED ISSUE #6: Update the header to match the style of tjcog.org

### PR DESCRIPTION
**Pull request by dskline**
Resolves #6

---

localhost_3000_(Laptop with HiDPI screen)

---

localhost_3000_(iPad)

localhost_3000_(iPad) (menu open)

---

localhost_3000_(iPhone 5_SE)

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
 
-import 'bootstrap/dist/css/bootstrap.min.css';
+import "./config/style/bootstrap.scss";
 
 ReactDOM.render(
   <React.StrictMode>
     <App />
   </React.StrictMode>,
-  document.getElementById('root')
+  document.getElementById("root")
```
Comment (dskline) [inline src/index.js:13]: I enabled eslint --fix on save, so there might be some tabbing and single quote changes here and there

**Review by dmeklund — APPROVED** [review]
This is really great, thanks a lot Spencer. @SudoTorres and @kmuenkel definitely recommend you take a look too. The swapping to the hamburger menu happens with the help of the bootstrap CSS classes? I suppose for someone that works a lot with Bootstrap that will be obvious but it might be worth a comment to explain what's going on there for the rest of us (and if there's anything else significant in the newly specified class names).

**Comment by dskline** [comment]
@dmeklund I'd be happy to leave more comments in the code than usual, if the gang wants to learn more about Bootstrap.  If your IDE allows it, you can usually CMD+click or Ctrl+click on a class name to see the associated styles attached.  Some are self explanatory but a lot are abbreviated (i.e. p-4 means padding: 1rem, and px-4 means padding: 1rem on the x-axis).

For the hamburger menu, in order to display/hide elements based on your screen size, you would normally write CSS like this:

```
// CSS
.menu-button-container {
    display: block
}
@media (min-width: 992px) {
    .menu-button-container {
        display: none;
    }
}

// JS
<div className="menu-button-container">
    <MenuButton />
</div>
```

But Bootstrap has some utility classes that allow you to shorten this to:

```
<div className="d-block d-lg-none">
    <MenuButton />
</div>
```

Here's [a link to Bootstrap's responsive design docs](https://v4-alpha.getbootstrap.com/layout/responsive-utilities/), but the tl;dr is that adding responsive classes are built on the "mobile first" idea.  In the case of the menu button, we want to display it on phone (xs and sm) and tablet (md) screens but hide it on desktops (lg and xl).

```
<div className="d-none" /> // --> hidden on xs screens and larger (i.e. everything)
<div className="d-md-none" /> // --> hidden on tablet screens and larger
<div className="d-lg-none" /> // --> hidden on desktop screens and larger

<div className="d-block d-lg-none" /> // --> display: block for xs and larger, but that will be overridden by the second class due to CSS specificity
```

The last one can be simplified to just `<div className="d-lg-none" />`, because the div component is display: block by default.

(anyway, I'm not sure how much we want to focus on mobile in this project, but I'm seeing fewer and fewer people using laptops so it might be a good skill to have)
